storage/src/test/scala/io/delta/storage/integration/S3LogStoreUtilIntegrationTest.scala

```diff
@@ -89,6 +89,19 @@ class S3LogStoreUtilIntegrationTest extends AnyFunSuite {
     })
   }
 
+  def testNonRecursive(table: String): Unit = integrationTest(table) {
+    // Setup delta log
+    touch(s"$testRunUID/$table/_delta_log/%020d.json".format(1))
+    // Setup data file
+    touch(s"$testRunUID/$table/1.snappy.parquet")
+    val resolvedPath = new Path(s"s3a://$bucket/$testRunUID/$table", "\u0000")
+    val ListByStartAfter =
+      S3LogStoreUtil.s3ListFromArray(fs, resolvedPath, resolvedPath.getParent)
+    val list = fs.listStatus(resolvedPath.getParent)
```

Comment:
can you explain to me what's going on / is being tested here?

Reply:
Ensure that the output of `S3LogStoreUtil.s3ListFromArray()` does not contain log files, which is the same as `fs.listStatus()`.

Reply:
Can you add these comments to the code? This is super helpful to understand the test and why it tests what we want!

Reply:
OK，done.